examples/chart/teleport-kube-agent/values.yaml

```diff
@@ -99,6 +99,67 @@ databases: []
 #      "*": "*"
 databaseResources: []
 
+################################################################
+# Values that must be provided if Windows Desktop access is enabled.
+################################################################
+
+# AD or non-AD configuration
+
+# (optional) Determines whether desktop sessions will show a user-selected wallpaper
+# vs a system-default, single-color wallpaper. For improved visual performance,
+# set this to false (its default value).
+#showWindowsDesktopWallpaper: false
```

Comment:
```suggestion
#windowsDesktopShowWallpaper: false
```

Perhaps more of a philosophical discussion, but I'd be tempted to make all of this a `windowsDesktop` section and just copy the section verbatim instead. I know it's not exactly the way that the chart works now, but this is an opportunity for us to influence future behaviour a bit. What do you think @hugoShaka?

```yaml
windowsDesktop:
  ldap:
    addr: '172.0.0.1:636'
    domain: 'asteroid.example.com'
    username: 'ASTEROID\svc-teleport'
    sid: 'S-1-5-21-1717630186-822870621-3846248122-1103'
    server_name: 'EC2AMAZ-ULM3Q3S.asteroid.example.com'
    insecure_skip_verify: false
    ldap_ca_cert: |
      -----BEGIN CERTIFICATE-----
      MIIDszCCApugAwIBAgIQK9/a6A9HNbFDkyqMrJFZ8TANBgkqhkiG9w0BAQwFADBs
      MRMwEQYKCZImiZPyLGQBGRYDY29tMRcwFQYKCZImiZPyLGQBGRYHZXhhbXBsZTEX
      MBUGCgmSJomT8ixkARkWB2FzdHJvaWQxIzAhBgNVBAMTGmFzdHJvaWQtRUMyQU1B
      Wi1VTE0zUTNTLUNBMB4XDTIzMDgyNjAxNDQyMloXDTI4MDgyNjAxNTQyMFowbDET
      MBEGCgmSJomT8ixkARkWA2NvbTEXMBUGCgmSJomT8ixkARkWB2V4YW1wbGUxFzAV
      BgoJkiaJk/IsZAEZFgdhc3Ryb2lkMSMwIQYDVQQDExphc3Ryb2lkLUVDMkFNQVot
      VUxNM1EzUy1DQTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAObKiE6G
      8K2RiIZUZOAZ0lP4fFx619pdhfdXtJ+JuFqPKmuWRgOoLMNudyeW3X35rZqkyI74
      TifjCe/Y0KkO1ix6sQQp3JXR0J+3cjF5IP+WXNqRBfrHvOMjadFboA8wgE8l9Fq0
      a4FdSCNkBNmftRO4vqvgtvdhsaLyexjjsyKTuYtvKwLcW8CQHCU6zGsx94ZaOswR
      nxB3VFY5kihkAnGu/pFh7oYaxmxrGQgtvnhD7PqsTnhRWQZRLyQCRpzjHikBLD5X
      1qBa9tJgEhOyQOBkCNJ6zwB4t7hyWcKf7cKb/pG3A8RxMJ66yWeywKAHMbRr9xUp
      iYClbN6ztvrD100CAwEAAaNRME8wCwYDVR0PBAQDAgGGMA8GA1UdEwEB/wQFMAMB
      Af8wHQYDVR0OBBYEFFGrFo9TdGzbvMbUxpqMUhA/H4tgMBAGCSsGAQQBgjcVAQQD
      AgEAMA0GCSqGSIb3DQEBDAUAA4IBAQBAhAKftgv7XHJEIXUQWp9HRA950znxHNGZ
      E7vNMccwwap1q5yds86xny+qrbkMseTaJLJxxQHMB6/7Xkmtjc+Ltsw/8VCOzHWA
      5nNF++tyllTLWBzypR3Ax68V3GYwnVbRIwOGiHrb1Y5MPzKsJF9h02U9ZehP2mwB
      o1tHYEmHbofA/plVRr+lBkY1lK0plmhfyqF4AuJzjhAMvwdkpZg/RGG6LjOEGeCX
      cuXopj9f2j+oYPAz798A1rZAm9Kz4b5uI46ultSVhU9WmM+nyjPU/J803J7sgy/m
      jwMCMyy8p33w+Y11a0av4eD/V0KMQdS90uxsUD5t1NTr2Fn5rEbg
      -----END CERTIFICATE-----
  discovery:
    base_dn: "*"
  labels:
    teleport.internal/resource-id: 5840df37-4cc3-49f1-ade0-ec9232575dbf
```

Reply:
It works unless you want to do "smart" things like secret mounting for the CA cert. I don't have a really strong opinion about this. Maybe I'd be a bit in favour of being smart here, as users can already pass everything through the override.

This also means we cannot guarantee more compatibility through the chart, I don't think that's a blocker.